### PR TITLE
AB#5017

### PR DESCRIPTION
**Pull request by sachin755**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by johnjdl** [comment]
@r-joyce please review and merge once the other priority PRs are done and development is bumped. These (5015 and 5017) are lower priorities.

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/assessment_platform/EntitiesAssessmentPlatformsDeletion.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/assessment_platform/EntitiesAssessmentPlatformsDeletion.js]: Should it be "Role" here?

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/assessment_platform/EntityAssessmentPlatformDetails.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/assessment_platform/EntityAssessmentPlatformDetails.js]: Hard-coded TLP value

**Review by r-joyce — CHANGES_REQUESTED** [review]
@johnjdl @kevinh4 Need some minor changes for hard-coded values

**Comment by johnjdl** [comment]
@r-joyce looks like Secutree made the fixes you requested. Go ahead and review again and merge when you have a moment.

**Comment by johnjdl** [comment]
@r-joyce Let's hold off on this 5015 and 5019 since it looks like there are some queries provided by Paul that Secutree will need to include.

**Review by r-joyce — APPROVED** [review]
Looks good to me 👍

**Comment by johnjdl** [comment]
@r-joyce go ahead and merge along with 5014. Please DO NOT merge 5019, it needs work still and has been sent back to active.

**Comment by johnjdl** [comment]
@r-joyce please merge this along with 5014 and 5019.

**Comment by johnjdl** [comment]
Nevermind @r-joyce please hold on merging 5014, 5017, and 5019 for now.

**Comment by johnjdl** [comment]
@r-joyce please review and merge these first today.

5014 - Data Entity: Parties
5019 - Data Entity: Locations
*5017 - Data Entity: Assessment Platform
5195 - Disable merge button

**Pull request merged**
